src/main/scala/com/databricks/labs/overwatch/validation/DeploymentValidation.scala

```diff
@@ -98,23 +98,29 @@ object DeploymentValidation extends SparkSessionWrapper {
     }
   }
 
-  private def validateMountCount(conf: MultiWorkspaceConfig): DeploymentValidationReport = {
+  private def validateMountMappingPath(conf: MultiWorkspaceConfig): DeploymentValidationReport = {
 
-    val isAzure = conf.cloud.toLowerCase == "azure" //Mount-point validation is only done for Azure
-    val isRemoteWorkspace = conf.workspace_id.trim != Initializer.getOrgId // No need to perform mount-point validation for driver workspace.
+    val path = conf.mount_mapping_path.trim
+    val testDetails =
+      s"""WorkSpaceMountTest
+         |mount_mapping_path:${path}
+         """.stripMargin
+    try {
+      if (!Helpers.pathExists(path)) {
+        DeploymentValidationReport(false,
+          getSimpleMsg("Validate_Mount"),
+          testDetails,
+          Some("Unable to find the provided csv: " + path),
+          Some(conf.workspace_id)
+        )
+      } else {
+        val inputDf = spark.read.option("header", "true")
+          .option("ignoreLeadingWhiteSpace", true)
+          .option("ignoreTrailingWhiteSpace", true)
+          .csv(path)
+          .select("mountPoint", "source")
```

Comment:
shouldn't we filter where organization_id == organization_id in the context?

```scala
.filter('workspaceId === conf.workspace_id)
```

Reply:
here we dont have workspaceId in mount_mapping_path csv file.